### PR TITLE
fixed args prefix for selenium3

### LINKED ISSUE #616: Custom Port values not adhered to - always looking for node on 4444
## Expected Behavior
Change capabilities host and port to custom values as: 

```
  capabilities: [{
        maxInstances: 5,
 
        browserName: 'chrome',
        acceptInsecureCerts: true,

        hostname: '127.0.0.1',
        port: 4449,

        'goog:chromeOptions': {
            args: [
                '--disable-gpu',
                '--no-sandbox',
                '--disable-infobars',
                 '--headless',
                '--disable-features=VizDisplayCompositor',
                '--window-size=1920,1080',
            ]
        }
    }],
```

Services as:

```
    services: [
        ['selenium-standalone', {
            logPath: './temp',
            installArgs: {
                version: "3.141.59",
                drivers: {
                    chrome: { version: "97.0.4692.71" }
                }
            },
            args: {
                version: "3.141.59",
                seleniumArgs: ['-host', '127.0.0.1', '-port', '4449'],
                drivers: {
                    chrome: { version: "97.0.4692.71" },
                }
            },
        }]
    ] 
```
Expect standalone to create hub/node for 4444 and run tests.

## Current Behavior

Get the following regardless of port number:

`
Failed to connect to selenium. Attempts left: 25 
 connect ECONNREFUSED 127.0.0.1:4444
`

## Steps to Reproduce
Provide an unambiguous set of steps to reproduce this bug. Include commands that you used
in the shell, or snippets of how you are using the JavaScript API.

https://github.com/abdul-farhat/wdio-7-example

https://gitpod.io/#https://github.com/abdul-farhat/wdio-7-example

```
npm i

npm run e2e-chrome

```

### PR DESCRIPTION
**Pull request by ArminazK**
fixes #616

### CODE REVIEW

File: lib/get-selenium-status-url.js [hunk 1]
```diff
@@ -46,8 +46,15 @@ exports.getRunningProcessType = function (seleniumArgs) {
  */
 exports.getSeleniumStatusUrl = function (seleniumArgs, opts) {
   const nodeConfigArg = seleniumArgs.indexOf('-nodeConfig');
-  const portArg = seleniumArgs.indexOf('--port');
-  const hostArg = seleniumArgs.indexOf('--host');
+
+// args prefix differs for selenium3 and selenium4
```
Comment (christian-bromann) [inline lib/get-selenium-status-url.js:50]:
```suggestion
  // args prefix differs for selenium3 and selenium4
```

**Review by christian-bromann — COMMENTED** [review]
Can we make sure to have a test for this? I wonder if it makes sense to add an e2e test suite to ensure that the whole process works.

**Comment by ArminazK** [comment]
> Can we make sure to have a test for this? I wonder if it makes sense to add an e2e test suite to ensure that the whole process works.

Hi, I was thinking about e2e tests. Seems that current tests target selenium 4 since version 8>.  In that case we need to have 2 tracks of tests (I mean not only for '--port' argument) ... This PR change works now on both versions and does not break current tests.

**Comment by christian-bromann** [comment]
@ArminazK cheers! There seems to be a couple of broken tests in CI. Can you take a look?

**Comment by christian-bromann** [comment]
@ArminazK ping

**Pull request closed without merging**